index.html

```diff
@@ -125,7 +125,7 @@ <h4>LET'S CONNECT</h4>
      <section class="tarjetas">
 
 
-         <div class="trabajo tamaño flex-column ">
+         <main class="trabajo tamaño flex-column ">
```

Comment:
I suggest you change the <main> tag to <section> as it is recommended by good practice for semantic structure, accessibility, styling, organization and maintainability of the code.